BossMod/Components/CastHint.cs

```diff
@@ -36,4 +49,78 @@ public override void OnCastFinished(BossModule module, Actor caster, ActorCastIn
                 _casters.Remove(caster);
         }
     }
+
+    public class CastInterruptHint : CastCounter
```

Comment:
Why isn't it a subclass of CastHint?

Reply:
I can do that, but then there needs to be an additional Hint string in the constructor it seems